api/v1/swagger.yaml

```diff
@@ -3614,9 +3635,20 @@ paths:
                 type: object
                 description: >-
                   Variables accessible inside the transform function.
+              generatorTemplate:
+                type: object
+                description: Details of Generator Template.
+                properties:
+                  name:
+                    type: string
+                    description: The Generator Template's name.
+                  version:
+                    type: string
+                    description: The Generator Template's version.
```

Comment:
description should be:

`A Generator Template version number (MAJOR.MINOR.PATCH) or a valid x-range, tilde range or caret range. Any of X, x, or * may be used to "stand in" for one of the numeric major, minor or patch values. A tilde range allows patch-level changes if a minor version is specified on the comparator, or allows minor-level changes if not. A caret range allows changes that do not modify the left-most non-zero major, minor or patch number. See https://github.com/npm/node-semver for more details on semver matching.`